pkg/codegen/python/gen.go

```diff
@@ -2130,8 +2127,8 @@ func (mod *modContext) genEnums(w io.Writer, enums []*schema.EnumType) error {
 	// Header
 	mod.genHeader(w, false /*needsSDK*/, nil)
 
+	fmt.Fprintf(w, "import pulumi\n")
 	// Enum import
-	fmt.Fprintf(w, "import builtins\n")
```

Comment:
`mod.genHeader` already adds `import builtins` (since https://github.com/pulumi/pulumi/pull/19035). The double import is harmless, but not noisy.